app/javascript/styles/_search.scss

```
@@ -325,7 +325,8 @@ nav.search-links, #search-panel {
 .advanced-opts {
   vertical-align: top;
   display: inline-block;
-  padding-top: 0.65em;
+  padding: 0.65em 0.25em;
+  border-radius: 1em;
 }
 
 .new-feature {
```

The "BETA" mark is too close to the preceding "Advanced search" text.  Also (and I don't feel too strongly about this) having the mark vertically extend above the text looks odd to me.

```suggestion
.new-feature {
  margin-left: 0.4em;
  vertical-align: sub;
```